pkg/commands/compute/compute_unit_test.go

```diff
@@ -83,78 +83,54 @@ func getFlags(cmd *kingpin.CmdClause) reflect.Value {
 }
 
 func TestCreatePackageArchive(t *testing.T) {
-	for _, testcase := range []struct {
```

Comment:
The changes to this function can be summarised as follows...

Remove the 'table test' structure, as it's redundant for a single test scenario. It just makes the code harder to read.